bin/internal/update_engine_version.sh

```diff
@@ -60,6 +61,8 @@ if [ -z "$ENGINE_VERSION" ] && [ -f "$FLUTTER_ROOT/DEPS" ] && [ -f "$FLUTTER_ROO
 fi
 
 # Write the engine version out so downstream tools know what to look for.
+echo $ENGINE_VERSION > "$FLUTTER_ROOT/bin/cache/engine.stamp"
```

Comment:
While not the same, we do use "engine_stamp" in this location:

```
SET engine_stamp=%cache_dir%\engine-dart-sdk.stamp
```

Reply:
Yeah, I think this should get cleaned up, this value is always the same.